flang/test/Analysis/AliasAnalysis/alias-analysis-5.fir

```diff
@@ -21,7 +21,7 @@ func.func @_QPtest1(%arg0: !fir.box<!fir.array<?xi32>> {fir.bindc_name = "arraya
   %1 = fir.load %arg3 : !fir.ref<i32>
   %2 = fir.convert %1 : (i32) -> index
   %3 = fir.convert %c1 : (index) -> i32
-  %4:2 = fir.do_loop %arg4 = %c1 to %2 step %c1 iter_args(%arg5 = %3) -> (index, i32) {
+  %4:2 = fir.do_loop %arg4 = %c1 to %2 step %c1 iter_args(%arg5 = %3) -> (index, i32) attributes {operandSegmentSizes = array<i32: 1, 1, 1, 1, 0>} {
```

Comment:
It shouldn't be necessary to print the operandSegment attribute in non generic form.